tests/installation/hostname_inst.pm

```diff
@@ -18,18 +18,22 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 use testapi;
+use version_utils;
 
 sub run {
     assert_screen "before-package-selection";
     select_console 'install-shell';
-    if (my $expected_install_hostname = get_var('EXPECTED_INSTALL_HOSTNAME')) {
```

Comment:
since you get rid of this variable, please remove it from `variables.md` and add the `NICTYPE_USER_OPTIONS` (which already exists in another test module but is still missing from `variables.md`